.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: go
+
+go:
+  - 1.8.x
```

Comment:
You can add the push to docker hub here?

Have a look how I've done it here https://github.com/MarketReaction/sentiment-go